manual/afqmc.tex

```diff
@@ -0,0 +1,285 @@
+\item \textbf{cutoff\_decomposition}. Cutoff used to stop the iterative cycle in the generation of the Cholesky decomposition of the 2-electron integrals. The generation of Cholesky vectors is stopped when the maximum error in the diagonal reaches this value. In case of an eigenvalue factorization, this becomes the cutoff applied to the eigenvalues. Only eigenvalues above this value are kept. Default: 1e-6
+\item \textbf{hdf\_write\_file}. If a value is given, an hdf5 restart file for the \texttt{Hamiltonian} is generated. Default: ``'' (no file is generated)
+\item \textbf{hdf\_write\_type}. What to store in the restart file. Options: ``integrals'': write directly the 1- and 2-electron integrals; ``factorized'': write the cholesky decomposition of the integrals; ``default'': Default value, write the type found in filetype.
+\item \textbf{ascii\_write\_file} If a value is given, an ASCII restart file for the \texttt{Hamiltonian} is generated. Default: ``'' (no file is generated)
+\item \textbf{nblocks}. This parameter controls the distribution of the 2-electron integrals among processors. In the default behavior (nblocks=1), all nodes contain the entire list of integrals. If nblocks $>$ 1, the of nodes in the calculation will be split in nblocks groups. Each node in a given group contains the same subset of integrals and subsequently operates on this subset during  any further operation that requires the hamiltonian. The maximum number of groups is NMO. Currently only works for filetype=``hdf5'' and the file must contain integrals.  Not yet implemented for input hamiltonians in the form of Cholesky vectors or for ASCII input. Coming soon!
+    Default: No distribution
+\item \textbf{printEig}. If ``yes'', prints additional information during the Cholesky decomposition.
+    Default: no
+\item \textbf{fix\_2eint}.  If this is set to ``yes'', orbital pairs that are found not to be positive definite are ignored in the generation of the Cholesky factorization. This is necessary if the 2-electron integrals are not positive definite due to round-off errors in their generation.
+    Default: no \\
+\end{itemize}
+
+\texttt{Wavefunction}: controls the object that manages the trial wave-functions. This block expects a list of xml-blocks defining actual trial-wave functions for various roles. 
+\texttt{<Wavefunction name="wfn0" info="info0">}
+  \texttt{<ImpSamp name="impsamp0" type="PureSD">}
+\begin{itemize}
+\item \textbf{init}. Argument in ImpSamp. Selects pre-defined trial wave-functions if desired. 
+Value: ground. Only for PureSD. Sets the trial wave-function to a block diagonal form. Diagonal in the occupied-occupied block and zero outside. This option corresponds to b
+tate determinant (assuming orbitals ordered by increasing eigenvalue) in the orbital space that defines the hamiltonian. For a \texttt{Hamiltonian} generates based on HF orbitals, this generates a HF trial wave-function. (Only useful situation.)
```

Comment:
This word at the line break shows up as 'b tate'.  What is this supposed to say?